spec.html

```diff
@@ -2021,7 +2022,7 @@ <h1>BinaryXor ( _x_, _y_ )</h1>
           <h1>BigIntBitwiseOp ( _op_, _x_, _y_ )</h1>
           <p>The abstract operation BigIntBitwiseOp takes arguments _op_, _x_, and _y_. It performs the following steps when called:</p>
           <emu-alg>
-            1. Assert: _op_ is *"&amp;"*, *"|"*, or *"^"*.
+            1. Assert: _op_ is ~&amp;~, ~|~, or ~^~.
```

Comment:
In the subsequent tests on `_op_`, you need to change the value that it's compared to.

(Alternatively, maybe BigIntBitwiseOp can just talk about `the result of applying _op_ to ...`, the same as NumberBitwiseOp and ByteListBitwiseOp do. And drop BinaryAnd, BinaryOr, BinaryXor.)